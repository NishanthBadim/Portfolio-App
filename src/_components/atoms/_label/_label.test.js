import React from "react";
import { create } from "react-test-renderer";
import { render, screen, getByRole } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import Label from "./_label";

describe("Label  testing", () => {
  test("Matches the snapshot", () => {
    const input = create(<Label />);
    expect(input.toJSON()).toMatchSnapshot();
  });

  test(" Label is present", () => {
    const {getByText}=render(<Label text={"label"}/>);
    const input = getByText("label");
        expect(input).toBeInTheDocument();
  });
});