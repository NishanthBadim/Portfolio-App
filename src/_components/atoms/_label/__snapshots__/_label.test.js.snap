// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label  testing Matches the snapshot 1`] = `
<div
  className="makeStyles-root-1"
>
  <p
    className="MuiTypography-root MuiTypography-body2"
  />
</div>
`;
